### PR TITLE
Fixed #32520 - Add review details to job wizard

### PR DESCRIPTION
**Pull request by MariaAga**
Screenshot from 2021-10-20 09-53-25
Screenshot from 2021-10-20 09-51-37

### CODE REVIEW

File: webpack/JobWizard/steps/ReviewDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    return `${hosts[0]}, ${hosts[1]} ${sprintf(
+      __(', and %s more'),
+      hostsCount - 2
+    )}`;
+  };
+  const [isAdvancedShown, setIsAdvancedShown] = useState(false);
+  const detailsFirstHalf = [
+    { label: __('Job Category'), value: jobCategory },
+    { label: __('Job template'), value: jobTemplate },
+    { label: __('Target hosts'), value: stringHosts() },
```
Comment (laviro) [inline webpack/JobWizard/steps/ReviewDetails/index.js:77]: Can we use the same collapse functionality as `Advanced fields`
if the hosts lists will be long, it's pretty hard to see the affected hosts:
Screenshot (39)

**Review by laviro — COMMENTED** [review]
A very clean PR, thanks @MariaAga,
only one comment

File: webpack/JobWizard/steps/ReviewDetails/index.js (hunk 1 above)
Comment (laviro) [reply webpack/JobWizard/steps/ReviewDetails/index.js:77]: I think as a summary, user should not go back to the hosts step. We could rather reuse the hosts modal in the summary: https://github.com/theforeman/foreman_remote_execution/pull/611#discussion_r731007851

Comment (laviro) [reply webpack/JobWizard/steps/ReviewDetails/index.js:77]: I have no issue with it being a number, although currently it's a text with tooltip... but let's allow clicking on it and expand the section like in the advanced fields section where we also don't need to return to the advanced wisard's step.

If not expanding, we could show the same hosts modal as in the hosts wizard step.

**Comment by MariaAga** [comment]
Thanks @adamruzicka 
1. I guess we shouldn't do that, so removed passwords from preview
2. https://github.com/theforeman/foreman_remote_execution/pull/658 fixed that :)

**Comment by adamruzicka** [comment]
Needs a rebase

**Comment by MariaAga** [comment]
rebased

**Review by adamruzicka — APPROVED** [review]
Looks good, let's get this in a revisit it later if needed

**Comment by adamruzicka** [comment]
Thank you @MariaAga & @Ron-Lavi !

**Pull request merged**